src/core/execute-serialize.c

```diff
@@ -3373,7 +3347,8 @@ static int exec_context_deserialize(ExecContext *c, FILE *f) {
                         if (val[0] == '-') {
                                 c->apparmor_profile_ignore = true;
                                 val++;
-                        }
+                        } else
+                                c->apparmor_profile_ignore = true;
```

Comment:
@YHNdnzj should this not be false?

Reply:
Oops, thank you for spotting this. Can you open a PR? (Or I'll fix it later)

Reply:
I'd prefer to leave it to you!